services/schedulable_transaction_body.proto

```diff
@@ -289,5 +293,20 @@ message SchedulableTransactionBody {
      * Update the metadata of one or more NFT's of a specific token type.
      */
     TokenUpdateNftsTransactionBody token_update_nfts = 41;
+
+    /**
+     * Transaction body for a scheduled transaction to create a new node.
+     */
+    NodeCreateTransactionBody nodeCreate = 53;
+
+    /**
+     * Transaction body for a scheduled transaction to modify an existing node.
+     */
+    NodeUpdateTransactionBody nodeUpdate = 54;
+
+    /**
+     * Transaction body for a scheduled transaction to remove a node.
+     */
+    NodeDeleteTransactionBody nodeDelete = 55;
```

Comment:
The rest of the file seems to be mostly alphabetically sorted and not numerically sorted, so we should adjust that.

Reply:
It's not all sorted alphabetically.  So kept them at the end.